examples/UberSDK/UberSDKTests/UberCore/KeychainUtilityTests.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  KeychainUtility.swift
+//  UberAuth
+//
+//  Copyright © 2024 Uber Technologies, Inc. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+
+import XCTest
+@testable import UberCore
+
+final class KeychainUtilityTests: XCTestCase {
+    
+    func test_save() {
+        let testObject = TestObject(
+            name: "test",
+            value: 5
+        )
+        
+        let keychainUtility = KeychainUtility()
```

Comment:
IIUC this is accessing the keychain in unit tests. Not sure if we should be doing that. Won't this leave keychain data in the test device which could interfere in the tests?

Reply:
Updated the test class to clear the test token on setUp and tearDown